utils/vlog/tests/carry/golden.pb_type.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="CARRY" num_pb="1">
+  <input name="CIN" num_pins="1"/>
+  <input name="I0" num_pins="4"/>
+  <input name="I1" num_pins="4"/>
+  <output name="COUT" num_pins="1"/>
```

Comment:
Missing `<pack_pattern type="carry"/>`;
```xml
  <output name="COUT" num_pins="1"/>
    <pack_pattern name="C" type="carry"/>
  </output>
```

This should be implicitly discovered because the `carry` attribute from the `CBLOCK` object should be propagated outwards.